lib/board.rb

```diff
@@ -0,0 +1,61 @@
+class Board
+	attr_accessor :cells
+
+	def initialize
+		reset!
+	end
+
+	 def reset!
+	 	@cells = Array.new(9, " ")
+	 end
+
+
+		
+
+	def display
+	    cell = "   "
+	    separator = "-----------"
+	    row1 = " #{cells[0]} " + "|" + " #{cells[1]} " + "|" + " #{cells[2]} "
+	    row2 = " #{cells[3]} " + "|" + " #{cells[4]} " + "|" + " #{cells[5]} "
+	    row3 = " #{cells[6]} " + "|" + " #{cells[7]} " + "|" + " #{cells[8]} "
+	    puts row1
+	    puts separator
+	    puts row2
+	    puts separator
+	    puts row3
+	end
+
+	def position(user_input)
+		cells[user_input.to_i-1]
+	end
+
+	def full?
+		cells.all? {|spot| spot == "X" || spot == "O"}
+	end
+
+	def turn_count
+		cells.count {|spot| spot == "X" || spot == "O"}
+	end
+
+	def taken?(user_input)
+		if cells[user_input.to_i-1] == "X"
```

Comment:
You should be able to use the implicit return value of the boolean expressions rather than returning true or false explicitly.

`cells[user_input.to_i-1] == "X" || cells[user_input.to_i-1] == "O"`